mlrun/track/trackers/mlflow_tracker.py

```diff
@@ -0,0 +1,552 @@
+# Copyright 2018 Iguazio
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import os
+import pathlib
+import tempfile
+import zipfile
+from typing import List, Union
+
+import mlflow
+import mlflow.entities
+import mlflow.environment_variables
+import mlflow.types
+
+import mlrun
+from mlrun import MLClientCtx, mlconf
+from mlrun.artifacts import Artifact, ModelArtifact
+from mlrun.features import Feature
+from mlrun.launcher.client import ClientBaseLauncher
+from mlrun.model import RunObject
+from mlrun.projects import MlrunProject
+from mlrun.utils import logger, now_date
+
+from ..tracker import Tracker
+
+
+class MLFlowTracker(Tracker):
+    """
+    A class for detecting and logging MLFlow runs into MLRun. Notice, only the last active MLFlow run is logged.
+    """
+
+    def __init__(self):
+        super().__init__()
+        self._pre_runs = None
+        self._experiment_name = None
+        self._run_id = None
+
+    @staticmethod
+    def is_enabled() -> bool:
+        """
+        Checks if tracker is enabled.
+
+        :return: True if the tracking configuration is enabled, False otherwise.
+        """
+        return getattr(mlconf.external_platform_tracking, mlflow.__name__).enabled
```

Comment:
I don't see why you need to use getattr (same below)
```suggestion
        return mlconf.external_platform_tracking.mlflow.enabled
```